### DISCUSSION TITLE
Data visualisation features

### DISCUSSION THREAD

**Discussion by MonicaGalassi** [MukurtuCMS/Mukurtu-CMS#30] [New Feature Request]
@michael-wynne-wsu I am aware some users used timelines from external sites and embedded in Mukurtu with great results! I was thinking that an integrated timeline would be very cool, especially if it's linkable to DHI and persona records. A tag cloud and a couple of basic infographics would be a great feature as well

**Comment by michael-wynne-wsu** [comment]
This would be a pretty new set of features. I'll make sure it's on the roadmap, but not something we'll likely aim for at launch.
